canu/utils/sls.py

```diff
@@ -410,7 +410,7 @@ def remote_request(
         sls_cache = None
 
         if os.getenv("SLS_API_GW") is None:
-            sls_url = "http://cray-sls.services.svc.cluster.local/v1/hardware"
+            sls_url = "https://cray-sls.services.svc.cluster.local/v1/hardware"
```

Comment:
same as above.  won't work in-mesh

Reply:
add a comment that the "if" side of this is in-mesh and the else is out of mesh because this is confusing from before your change.

Reply:
Fixed